src/components/utils/__tests__/formatValue.spec.ts

```diff
@@ -27,7 +27,7 @@ describe('formatValue', () => {
     ).toEqual('1.234.567');
   });
 
-  it('should handle comma separator for decimals', () => {
+  it.skip('should handle comma separator for decimals', () => {
```

Comment:
comma-separated decimal has an `.` for `decimalSeparator`
 If it's an expected code please drop a comment it's misleading